bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -7900,9 +7904,13 @@ public Report addReport(String localPath, String sourceModuleName, String report
 		// Figure out the create time of the report.
 		long createTime = 0;
 		try {
-			java.io.File tempFile = new java.io.File(localPath);
-			// Convert to UNIX epoch (seconds, not milliseconds).
-			createTime = tempFile.lastModified() / 1000;
+			if (localPath.toLowerCase().contains("http:")) {
+				createTime = System.currentTimeMillis() / 1000;
+			} else {
+                java.io.File tempFile = new java.io.File(localPath);
```

Comment:
Can you check your editor settings to make sure its inserting spaces instead of tabs?